github/Repository.py

```diff
@@ -2239,6 +2259,19 @@ def protect_branch(self, branch, enabled, enforcement_level=github.GithubObject.
             headers={'Accept': 'application/vnd.github.loki-preview+json'}
         )
 
+    def replace_topics(self, topics):
```

Comment:
I am not sure what you mean by checking the POST.  The API doesn't mention that being a valid call for either get or replace topics.

Reply:
I mean could you test the `PUT`, i.e this `replace_topics` method

Reply:
Agreed -- in my testing, it gave an error (see PR comments) if I passed back the same structure I got, but works if I put the hash as the value of the dict `{'names': ..}`. It would probably be easiest and more consistent